.goreleaser-nightly.yml

```diff
@@ -49,18 +49,21 @@ builds:
       pre: make generate
 
 nfpms:
-  -
-    id: "influxdb2"
+  - id: "influxdb2"
     builds: ["influx", "influxd"]
     formats:
       - deb
       - rpm
     bindir: /usr/bin
-    files:
-      "scripts/init.sh": "/usr/lib/influxdb/scripts/init.sh"
-      "scripts/influxdb.service": "/usr/lib/influxdb/scripts/influxdb.service"
-      "scripts/logrotate": "/etc/logrotate.d/influxdb"
-      "scripts/influxdb2-upgrade.sh": "/usr/share/influxdb/influxdb2-upgrade.sh"
+    contents:
```

Comment:
Collateral damage from the goreleaser version bump, the old form is now deprecated